### PR TITLE
Fix javatools

### PR DESCRIPTION
**Pull request by katre**

### CODE REVIEW

**Comment by katre** [comment]
buildjar is also not compatible with jdk8, so removed those tests on ubuntu1404 and ubuntu1604

File: src/java_tools/buildjar/javatests/com/google/devtools/build/java/turbine/javac/BUILD [hunk 1]
```diff
@@ -72,10 +64,6 @@ java_test(
     data = [
         "//third_party/java/jdk/langtools:javac_jar",
     ],
-    jvm_flags = [
-        # Simulates how Bazel invokes turbine
-        "-Xbootclasspath/p:$(location //third_party/java/jdk/langtools:javac_jar)",
```
Comment (iirina) [inline src/java_tools/buildjar/javatests/com/google/devtools/build/java/turbine/javac/BUILD]: Why did you remove these flags? They may be the reason why the buildtools tests are failing on jdk8

Comment (katre) [reply src/java_tools/buildjar/javatests/com/google/devtools/build/java/turbine/javac/BUILD]: Because the build fails with them on JDK11.

**Comment by berrylaw** [comment]
「katre <notifications@github.com>」在 2019年4月26日 週五，下午10:29 寫道：

File: src/java_tools/buildjar/javatests/com/google/devtools/build/java/turbine/javac/BUILD (hunk 1 above)
Comment (iirina) [reply src/java_tools/buildjar/javatests/com/google/devtools/build/java/turbine/javac/BUILD]: OK, I think it's ok to disable the tests for JDK8 for the time being... I'll investigate the issue  on Monday. Thanks!

**Comment by berrylaw** [comment]
Where

「bazel.build machine account <notifications@github.com>」在 2019年4月26日
週五，下午11:46 寫道：

File: src/java_tools/buildjar/javatests/com/google/devtools/build/java/turbine/javac/BUILD (hunk 1 above)
Comment (berrylaw) [reply src/java_tools/buildjar/javatests/com/google/devtools/build/java/turbine/javac/BUILD]: Where

**Pull request closed without merging**